awx/main/models/inventory.py

```diff
@@ -914,22 +914,7 @@ class InventorySourceOptions(BaseModel):
 
     injectors = dict()
 
-    SOURCE_CHOICES = [
-        ('file', _('File, Directory or Script')),
-        ('constructed', _('Template additional groups and hostvars at runtime')),
-        ('scm', _('Sourced from a Project')),
-        ('ec2', _('Amazon EC2')),
-        ('gce', _('Google Compute Engine')),
-        ('azure_rm', _('Microsoft Azure Resource Manager')),
-        ('vmware', _('VMware vCenter')),
-        ('satellite6', _('Red Hat Satellite 6')),
-        ('openstack', _('OpenStack')),
-        ('rhv', _('Red Hat Virtualization')),
-        ('controller', _('Red Hat Ansible Automation Platform')),
-        ('insights', _('Red Hat Insights')),
-        ('terraform', _('Terraform State')),
-        ('openshift_virtualization', _('OpenShift Virtualization')),
-    ]
+    SOURCE_CHOICES = []
 
```

Comment:
It doesn't seem to be used anywhere.
```suggestion
```

Reply:
I'm so sorry, Sviat- I didn't see this second comment with the option to commit the suggestion before I made the change! This has been resolved via this commit [973b591ac8bc6fad6cdabf0d5cb7a028c79dacb9](https://github.com/ansible/awx/pull/15537/commits/973b591ac8bc6fad6cdabf0d5cb7a028c79dacb9)

Reply:
Ack. Let's resolve this thread.